### DISCUSSION TITLE
Unexpected token error when blocking domains

### DISCUSSION THREAD

**Discussion by Calbo67** [spatie/browsershot#527] [Bugs]
My browsershot works fine when crawling websites but as soon as I add blockDomains i get unexpected token error.

![image](https://user-images.githubusercontent.com/29214275/116165215-4d5aeb80-a6f3-11eb-999f-734c62c79bb6.png)
![image](https://user-images.githubusercontent.com/29214275/116165267-66fc3300-a6f3-11eb-8f6c-f3062bf1c73e.png)

**Comment by freekmurze** [comment]
I don't have the time to look into this now, but I'd accept a PR that fixes this.

**Comment by clementmas** [comment]
I had the same issue. I proposed a PR here: https://github.com/spatie/browsershot/pull/564

Edit: PR got merged. It's fixed now
